src/Acuminator/Acuminator.Utilities/Roslyn/Semantic/PXGraph/Utils/GraphSymbolUtils.cs

```diff
@@ -40,11 +40,9 @@ public static INamedTypeSymbol GetGraphFromGraphExtension(this ITypeSymbol graph
 				return null;
 
 			ITypeSymbol firstTypeArg = graphExtTypeArgs.Last();
-
-			if (!(firstTypeArg is INamedTypeSymbol pxGraph) || !pxGraph.IsPXGraph())
-				return null;
-
-			return pxGraph;
+			return firstTypeArg.IsPXGraph()
+				? firstTypeArg
```

Comment:
When can `firstTypeArg` be non-`INamedTypeSymbol` type?

Reply:
@dropsonic in quite a number of cases when it is type parameter. It's almost any generic graph extension in Pure, for  example `ContactAddressGraph<TGraph>`

Reply:
Actually `IsPXGraph()` method is also not good enough, it needs to detect graph as a type parameter too